src/components/ThirdPartyCollectionDetailPage/ThirdPartyCollectionDetailPage.tsx

```diff
@@ -43,11 +104,51 @@ export default class ThirdPartyCollectionDetailPage extends React.PureComponent<
     return collection !== null && canSeeCollection(collection, wallet.address)
   }
 
+  areAllSelected(items: Item[]) {
+    const { itemSelectionState } = this.state
+    return items.every(item => itemSelectionState[item.id])
+  }
+
+  getSelectedItems() {
+    const { itemSelectionState } = this.state
+    return Object.values(itemSelectionState).filter(isSelected => isSelected)
+  }
```

Comment:
It might be better to have a method to return the amount of selected items as we're not using the result of this method for other thing than for counting how many items are selected.

Reply:
True!